client/src/containers/CreateCardForm/index.js

```diff
@@ -23,7 +23,7 @@ class CreateCardFormContainer extends Component {
   createCard = (event) => {
     event.preventDefault()
     api.cards.create(
-      { description: this.state.description },
+      { description: this.state.description, readiness: false },
```

Comment:
@nikitasakov thoughts about setting `completed_at` or similar to null as a default in database? Without passing it as a param